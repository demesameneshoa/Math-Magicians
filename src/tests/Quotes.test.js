import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import Quote from '../components/Quotes';

describe('Home component unit test', () => {
  test('Home Component Renders Properly ', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
