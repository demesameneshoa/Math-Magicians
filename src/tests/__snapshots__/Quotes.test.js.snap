// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component unit test Home Component Renders Properly  1`] = `
<div
  className="quotediv"
>
  <q
    className="quote"
  />
  <p
    className="p"
  />
</div>
`;
